src/main/java/Events.java

```diff
@@ -0,0 +1,19 @@
+import java.util.*;
+
+/**
+ * tasks that STARTS at a specific time and ENDS at a specific time.
+ */
+public class Events extends Task {
+
+    private String at;
```

Comment:
Is this variable name for the date/time that the event occurs too short? Perhaps make it a bit longer and more explicit like dateOfEvent ?